docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--org-section-long-form--default.html.twig

```diff
@@ -1,22 +1,32 @@
-{% set heading = { } %}
+{% set title = '' %}
 {% if paragraph.field_hide_heading is not defined or paragraph.field_hide_heading.0.value == 0 %}
-  {% set heading = {
-    "title": (content.field_section_long_form_heading|field_value),
-    "id": "",
-    "borderless": true
-  } %}
+  {% set title = content.field_section_long_form_heading|field_value %}
+{% endif %}
+
+{% set contained = false %}
+{% set paragraphType = content.field_section_long_form_content[0]["#paragraph"]["#values"]["type"] %}
+{% if paragraphType in ['iframe', 'caspio', 'image', 'tableau'] %}
+  {% set contained = true %}
 {% endif %}
 
 {% embed "@organisms/by-author/stacked-row-section.twig" with {
-  "stackedRowSection": heading,
+  "stackedRowSection": {
+    "title": title,
+    "id": "",
+    "borderless": true,
+    "contained": contained
+  },
   "stackedRow": {
     "borderless": true
-  },
-  "externalSidebar": true
+  }
 } %}
   {% block stackedRowContentOverride %}
     {{ content.field_section_long_form_content }}
   {% endblock %}
 {% endembed %}
 
 {{ content.field_section_long_form_addition }}
+
+{{ dd(content.field_section_long_form_content[0]["#paragraph"]) }}
+{{ dd(content.field_section_long_form_content[0]["#paragraph"]["#values"]) }}
+{{ dd(content.field_section_long_form_content[0]["#paragraph"]["#values"]["type"]) }}
```

Comment:
The problem with `dd()` is that it doesn't show which properties are protected, `["#values"]` is one of these and can't be accessed directly, instead you need to target each field or in this case if you want to know the type or bundle, you can do this:
`content.field_section_long_form_content[0]["#paragraph"].bundle()`